src/ensembl/src/shared/components/radio-group/RadioGroup.tsx

```diff
@@ -79,7 +71,10 @@ const RadioGroup = (props: Props) => {
     <div>
       {props.options.map((option, index) => (
         <div key={index} className={wrapperClass}>
-          <div onClick={() => handleOnChange(option.value)} className={getRadioClass(option)} />
+          <div
+            onClick={() => handleOnChange(option.value)}
```

Comment:
I missed this in Ridwan's PR. Is't perfectly fine to call a function `handleChange`; it's also fine to call it `onChange`; but `handleOnChange` sounds awkward. Will it be ok to rename this function?

Reply:
@azangru it looks like handleOnChange is doing something else and then calling onChange, have a look at line 44.  can we still change this to onChange?

Reply:
I think so. It's component's own onChange callback that does something and then calls the onChange that came with the props.